modules/miner/miner.go

```diff
@@ -165,7 +169,19 @@ func New(cs modules.ConsensusSet, tpool modules.TransactionPool, w modules.Walle
 		headerMem:  make([]types.BlockHeader, HeaderMemory),
 
 		fullSets:  make(map[modules.TransactionSetID][]int),
-		splitSets: make(map[int]*splitSet),
+		splitSets: make(map[splitSetID]*splitSet),
+		blockMapHeap: &mapHeap{
+			selectID: make(map[splitSetID]*mapElement),
+			data:     make([]*mapElement, 0),
+			size:     uint64(0),
```

Comment:
The `uint64` cast is implicit, and isn't needed. You can just set it to 0.

The zero is also implicit, and isn't needed. You can just omit this line altogether.